.travis.yml

```diff
@@ -31,11 +31,13 @@ notifications:
   email: false
 
 before_install:
+  - echo "extension=ldap.so" >> `php --ini | grep "Loaded Configuration" | sed -e "s|.*:\s*||"`
```

Comment:
Use travis config-add instead. See https://github.com/zendframework/zend-cache/blob/master/.travis.yml#L64 for an example